doc/user/load_save.md

```diff
@@ -813,6 +813,43 @@ std::cout << std::endl << std::endl;
 mlpack::data::Save("mlpack-favicon.png", image, info, true);
 ```
 
+### Letterbox transform
+
+The letterbox transform resizes an image's dimensions to `imgSize x imgSize` but
+keeps the aspect ratio of the original image. Whitespace is then filled in
+with `fillValue`.
+
+- `LetterboxImage(src, opt, imgSize, fillValue)`
+  * `src` is a [column-major matrix](matrices.md) containing a single image,
+    where the image is represented as a flattened vector in one column.
+  * `opt` is a [`data::imageOptions&`](#dataimageinfo) containing info on
+    the dimensions of the image.
+  * `imgSize` is a `const size_t` determining the new width and height of
+    `src`.
```

Comment:
Does it make sense to make the method more flexible and allow different width/height in the resized image?  I know YOLOv3Tiny only needs it as a letterboxed square, but maybe this would make it more flexible for other cases?  It feels like it should be easy enough to implement.